packages/apps-config/src/endpoints/testing.ts

```diff
@@ -19,10 +19,9 @@ export * from './testingRelayWestend';
 export const testChains: EndpointOption[] = [
   {
       info: 'matic',
-      isDevelopment: false,
-      text: 'Avail-Testnet',
+      text: 'Avail-Devnet 06',
       providers: {
-        'POLYGON': config.testnet_url
+        'POLYGON': 'wss://devnet06.dataavailability.link:28546'
```

Comment:
why do we need to change from config to manual insertion?

Reply:
As we will have several networks, like `devnetXX`, `tesnetXXX`, and `mainnet`, we will need to add each of them one by one in the respective `endpoints` (`testing.ts` & `production.ts`)